src/bootstrap/bin/rustdoc.rs

```diff
@@ -2,6 +2,7 @@
 //!
 //! See comments in `src/bootstrap/rustc.rs` for more information.
 
+use bootstrap::util::output;
```

Comment:
How does this work?
`src/bootstrap/bin/{rustc,rustdoc}.rs` shouldn't depend on bootstrap.
If `dylib_util.rs` is included into these files then it shouldn't contain code depending on bootstrap, so the `LLD_NO_THREADS` stuff should be moved to some other place.